tests/system/Validation/ValidationTest.php

```diff
@@ -665,24 +665,25 @@ public function testRulesForArrayField($body, $rules, $results)
 
 	//--------------------------------------------------------------------
 
-	public function arrayFieldDataProvider() {
+	public function arrayFieldDataProvider()
+	{
 		return [
 			'all_rules_should_pass' => [
-				'body' => [
+				'body'    => [
 					'foo' => [
 						'a',
 						'b',
-						'c'
-					]
+						'c',
+					],
 				],
-				'rules' => [
+				'rules'   => [
 					'foo.0' => 'required|alpha|max_length[2]',
 					'foo.1' => 'required|alpha|max_length[2]',
-					'foo.2' => 'required|alpha|max_length[2]'
+					'foo.2' => 'required|alpha|max_length[2]',
 				],
-				'results' => []
+				'results' => [],
 			],
-			'first_field_will_return_required_error' => [
+			/*'first_field_will_return_required_error' => [
```

Comment:
just wondering, why this data provider commented ?

Reply:
:man_facepalming: :man_facepalming: :man_facepalming: My bad. Will create another PR.

Thank you for pointing this out